js/src/xt.d.ts

```
@@ -0,0 +1,100 @@
+import Exchange from './abstract/xt.js';
+import { Currencies, Currency, Dict, FundingHistory, FundingRateHistory, Int, LeverageTier, MarginModification, Market, Num, OHLCV, Order, OrderSide, OrderType, Str, Tickers, Transaction, TransferEntry, LedgerEntry, FundingRate, DepositAddress, LeverageTiers } from './base/types.js';
+/**
+ * @class xt
+ * @augments Exchange
+ */
+export default class xt extends Exchange {
+    describe(): any;
+    nonce(): number;
+    fetchTime(params?: {}): Promise<number>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)